[BUG] react-formio fails on `prepublish` during `npm install`

## Environment

- **react-formio branch/tag:**
  - [x] master
  - [x] 1x
- **NodeJS version:** v12.7.0
- **npm version:** 6.10.0
- **Operating System:** Microsoft Windows 10 version 1903 (May 2019 Update)

## Steps to Reproduce

1. Clone `react-formio`
1. Run `npm install`

## Expected behavior
`npm install` should complete successfully.

## Observed behavior

`npm install` fails with the following warning and error messages on the console.

```console
npm WARN prepublish-on-install As of npm@5, `prepublish` scripts are deprecated.
npm WARN prepublish-on-install Use `prepare` for build steps and `prepublishOnly` for upload-only.
npm WARN prepublish-on-install See the deprecation note in `npm help scripts` for more information.
npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\sangh\AppData\Roaming\npm-cache\_logs\2019-07-31T03_23_01_083Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! react-formio@4.0.0-rc.11 prepublish: `npm run lint; npm run build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the react-formio@4.0.0-rc.11 prepublish script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\sangh\AppData\Roaming\npm-cache\_logs\2019-07-31T03_23_01_165Z-debug.log
```

--- a/package.json
+++ b/package.json
@@ -8,7 +8,7 @@
     "build": "babel ./src --ignore=*.spec.js --out-dir ./lib --presets es2015,react,stage-2",
     "lint": "eslint src",
     "watch": "babel ./src --watch --ignore=*.spec.js --out-dir ./lib --presets es2015,react,stage-2",
-    "prepublish": "npm run lint; npm run build"
+    "prepublish": "npm run lint && npm run build"
   },
   "repository": {
     "type": "git",

